src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-detail/cephfs-detail.component.ts

```diff
@@ -31,15 +33,19 @@ export class CephfsDetailComponent implements OnChanges, OnInit {
   clientCount: number;
   mdsCounters = {};
   grafanaId: any;
+  grafanaScope: Permission;
```

Comment:
Since we use `permission` in all other files, I would suggest:
```suggestion
  grafanaPermission: Permission;
```

Reply:
ack